index.html

```diff
@@ -32,10 +32,10 @@ <h3><!-- Student 2 Insert your name here --></h3>
 	  <p><!-- Student 2 insert some text here --></p>
 	</div>
 	<br>
-	<div class="cs3" id="student3">
-	  <img src=""><!-- Student 3 insert an image here and set it's width to 600px. Don't forget to add and alt text -->
-	  <h3><!-- Student 3 Insert your name here --></h3>
-	  <p><!-- Student 3 insert some text here --></p>
+	<div class="cs3" id="Sophie">
+	  <img src="images/skier.jpeg" alt="This is a skier">
```

Comment:
When inserting an image, you are supposed to start with "./[LINK]". So for example:  <img src="**./**images/skier.jpeg"